niworkflows/interfaces/tests/test_images.py

```diff
@@ -3,8 +3,44 @@
 import nibabel as nb
 from nipype.interfaces import nilearn as nl
 from .. import images as im
+from pathlib import Path
 
 import pytest
+qform = np.eye(4)
```

Comment:
```suggestion
```

Reply:
Ah yes--sorry! Editing on github rather than in a an editor tends to make me miss stuff--you were clear, I just missed it.